### PR TITLE
Revert loading migrations instead of publishing

### PR DESCRIPTION
**Pull request by promatik**
Fix to @ian-patel issue on https://github.com/Laravel-Backpack/PermissionManager/commit/e3b9756afc2b4eb0ca11b4770b0c20356cc50669#r50448573

This reverts the loading of the migrations, instead of publishing it.

@tabacitu I'm not sure if we should ever change this, probably in an upcoming version we can even remove this migration since it's only needed when migrating from 4.0 to 4.1.

### CODE REVIEW

**Comment by ian-patel** [comment]
Thank you @promatik.

I believe, upgrade is one-off thing, we should keep publishes then autoloading migration scripts. Devs can have more flexibility to customise the code.
And @tabacitu thank you for prioritize this issue, really appreciate it.

**Review by ian-patel — APPROVED** [review]

**Comment by tabacitu** [comment]
Great, thanks guys! And sorry for the bug @ian-patel .

Bottom line: version 6.0.7 is the only one affected by the migration loading bug. We will consider removing that patch release from packagist, but as of now... it seems safer to just tell people who get the "duplicated migrations" error to do a `composer update` and get 6.0.8. If we _did_ delete 6.0.7 we'd introduce a different bug for people who run 6.0.7... they will have problems running `composer install` since that version no longer exists.

TLDR: 6.0.8 good, 6.0.7 bad 😅
